clustintime/visualization.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Result visualization for clustintime
+"""
+
+# import networkx as nx
+# creation, manipulation and study of the structure, dynamics and functions of complex networks
+
+from matplotlib import patches
+import matplotlib.pyplot as plt  # For graphs
+import networkx as nx
+
+# Libraries
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from dyneusr import DyNeuGraph
+from dyneusr.mapper.utils import optimize_dbscan
+from IPython.display import HTML, display
+from kmapper import Cover, KeplerMapper
+from sklearn.decomposition import PCA
+from sklearn.manifold import TSNE
+from umap.umap_ import UMAP
+
+
+# def display_table(data):
+#     """
+#     HTML code for showing a table of the results
+
+#     Parameters
+#     ----------
+#     data : numpy array
+#     Returns
+#     -------
+#     None.
+
+#     """
+#     html = "<table>"
+#     for row in data:
+#         html += "<tr>"
+#         for field in row:
+#             html += f"<td>{field}<td>"
+#         html += "</tr>"
+#     html += "</table>"
+#     display(HTML(html))
```

Comment:
Consider deleting this code.
If we need to fetch this, we can always check back on commit history.